src/sentry/replays/lib/new_query/fields.py

```diff
@@ -64,7 +66,7 @@ def _apply_composite(self, expression: Expression, operator: str, value: list[T]
         elif operator == "NOT IN":
             visitor = self.query.visit_not_in
         else:
-            raise Exception(f"Unsupported composite search operator: '{operator}'")
+            raise OperatorNotSupported(f"Unsupported aggregate search operator: '{operator}'")
```

Comment:
why does this one change to `aggregate`?

Reply:
Aggregate seemed to be a more approachable phrasing.  I'll think some more about the language here.

Reply:
was just curious because the other ones all say `composite` -- didn't know if this one was different somehow

Reply:
Oh I must have forgotten to change the others.  I'll fix it or revert back to composite.